### DISCUSSION TITLE
Var option

### DISCUSSION THREAD

**Discussion by AbelChica** [balta-io/2801#60] [Perguntas e Respostas] [answered]
![image](https://user-images.githubusercontent.com/14608761/158884543-44ebb85c-5a85-4877-b54d-6406f26d55ad.png)
 Poderia me explicar melhor o por que desse var option?

Obrigado

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @AbelChica , como vai?

o `var` ali tem a mesma finalidade de um `short`, ele é o que chamamos de syntax sugar, apenas para facilitar a assimilação de tipos no C#.
Tem uma aula só sobre `var` no curso Fundamentos de C#, no módulo 5.

Em seguida a declaração do `var option` temos a assimilação do valor que o usuário irá digitar no console, isto é feito utilizando `Console.ReadLine()`.

Como o resultado de um `Console.ReadLine` é uma string, nós fazemos a conversão dela para `short` utilizando o `short.Parse`, como também mostrado no módulo 5 do curso de C#.

Desta forma, temos a opção que o usuário quer acessar no nosso sistema.
